collector/benchmarks/README.md

```diff
@@ -0,0 +1,87 @@
+# The Benchmark Suite
+
+This file describes the programs in the benchmark suite and explains why they
+were included.
+
+## Real programs that are important
+
+These are real programs that are important in some way, and worth tracking.
+
+- **cargo**: The Rust package manager. An important program in the Rust
+  ecosystem.
+- **clap-rs**: A command line argument parser. A crate used by many Rust
+  programs.
+- **cranelift-codegen**: The largest crate from a code generator. Used by
+  Firefox.
+- **crates.io**: Source code for `crates.io`. An important site in the
+  Rust ecosystem.
+- **futures**: A futures implementation. Used by many Rust programs.
+- **helloworld**: A trivial program. Gives a lower bound on compile time.
+- **hyper**: An HTTP library. An important part of the Rust ecosystem.
+- **piston-image**: A modular game engine. An interesting Rust program.
+- **regex**: A regular expression parser. Used by many Rust programs.
+- **ripgrep**: A line-oriented search tool. A widely-used utility.
+- **script-servo**: Servo's `script` crate. A particularly large crate.
+- **sentry-cli**: A command line utility.
+- **serde**: A serialization/deserialization crate. Used by many other
+  Rust programs.
+- **style-servo**: Servo's `style` crate. A large crate, and one used by
+  Firefox.
+- **syn**: A library for parsing Rust code. An important part of the Rust
+  ecosystem.
+- **tokio-webpush-simple**
```

Comment:
Maybe something like "futures-heavy crate" for reason why we're including it; not sure what to give for the description. Could just be "Simple web server with Tokio" or something to that effect (not sure if that's true).